src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -262,6 +312,66 @@ export default {
 
             this.time = this.timeFormat.format(datum);
             this.imageUrl = this.imageFormat.format(datum);
+        },
+        getFormatter(key) {
+            let valueFormatter = this.openmct.telemetry.getValueFormatter({
+                format: key
+            });
+
+            return valueFormatter.formatter;
+        },
+        numericImageAge() {
+            let currentTime = this.currentTimeValue();
+            let selectedImage = this.getSelectedImage();
+
+            if (selectedImage === undefined || selectedImage[this.timeKey] === undefined) {
+                return;
+            }
+
+            return currentTime - selectedImage[this.timeKey];
```

Comment:
Need to parse `selectedImage[this.timeKey]` using telemetry formatter. A lot of our backends provide times as strings, but the current time returned by our API will always be a number.

Reply:
Gotcha! I'll update.